src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -1085,6 +1108,11 @@ private PreparedStatement getUpdateStreamFromDb(Connection conn)
     GSA_TIMESTAMP;
   }
 
+  enum ResultSetNext {
+    CONTINUE_NEXT,
+    STOP_NEXT,
+  }
+
```

Comment:
How about
PROCESS_ALL_RESULTS
PROCESS_ONE_RESULT

Am I understanding the semantics?

Reply:
Thx, modified these.